pkg/cmd/dev/testlogic.go

```diff
@@ -127,11 +131,18 @@ func (d *dev) testlogic(cmd *cobra.Command, commandLine []string) error {
 		return err
 	}
 
-	if !noGen {
+	if forceGen {
```

Comment:
```
if !noGen {
    if forceGen || d.shouldGenerateLogicTests(ctx) {
        ...
    }
}
```

Just reduces some duplication to write the conditional this way. I suppose this is equivalent:

```
if !noGen && (forceGen || d.shouldGenerateLogicTests(ctx)) {
   ....
}
```